src/gui/modals/SetupDialog.cpp

```diff
@@ -1354,41 +1354,41 @@ void SetupDialog::setLADSPADir(const QString & ladspaDir)
 }
 
 
-void SetupDialog::openSF2Dir()
+void SetupDialog::openSoundFontDir()
 {
 	QString new_dir = FileDialog::getExistingDirectory(this,
-		tr("Choose your SF2 directory"), m_sf2Dir);
+		tr("Choose your SoundFont directory"), m_soundFontDir);
 	if (!new_dir.isEmpty())
 	{
-		m_sf2DirLineEdit->setText(new_dir);
+		m_soundFontDirLineEdit->setText(new_dir);
 	}
 }
 
 
-void SetupDialog::setSF2Dir(const QString & sf2Dir)
+void SetupDialog::setSoundFontDir(const QString & soundFontDir)
```

Comment:
(Non-blocking) Per the Coding Conventions [re: pointers](https://github.com/LMMS/lmms/wiki/Coding-Conventions#pointers):

> The `*` symbol in pointer declarations MUST be adjacent to the type, not the variable name. This rule also applies to `**`, `&`.

```suggestion
void SetupDialog::setSoundFontDir(const QString& soundFontDir)
```